src/sdk-reference/android/3/kuzzle/constructor/index.md

```diff
@@ -72,7 +72,7 @@ This is the main entry point to communicate with Kuzzle. Every other object inhe
 | ``queueTTL`` | integer | Time a queued request is kept during offline mode, in milliseconds | Yes |
 | ``replayInterval`` | integer | Delay between each replayed requests | Yes |
 | ``reconnectionDelay`` | integer | Number of milliseconds between reconnection attempts | No |
-| ``sslConnection`` | boolean | Connect to Kuzzle using SSL | No |
+| ``ssl`` | boolean | Connect to Kuzzle using SSL | No |
```

Comment:
I think we could use the same property name for the latest version of our SDKs.
We could have the 2 version of this property for the JS since it is already released.